src/main/java/org/folio/service/invoice/InvoiceLineService.java

```diff
@@ -93,7 +98,12 @@ public Future<Void> removeEncumbranceLinks(List<InvoiceLine> invoiceLines, List<
         }
       }
     }
-    return saveInvoiceLines(invoiceLinesToUpdate, requestContext);
+    if (CollectionUtils.isNotEmpty(invoiceLinesToUpdate)) {
+      logger.info("removeEncumbranceLinks:: updating invoice lines: {} with removed encumbrance links",
+        invoiceLinesToUpdate.stream().map(InvoiceLine::getId).collect(joining(", ")));
+      return saveInvoiceLines(invoiceLinesToUpdate, requestContext);
+    }
+    return Future.succeededFuture();
```

Comment:
very small suggestion. 
    if (CollectionUtils.isEmpty(invoiceLinesToUpdate)) {
      logger.info("removeEncumbranceLinks:: no invoice lines to update");
      return Future.succeededFuture();
    }
    logger.info("removeEncumbranceLinks:: updating invoice lines: {} with removed encumbrance links",
        invoiceLinesToUpdate.stream().map(InvoiceLine::getId).collect(joining(", ")));
    return saveInvoiceLines(invoiceLinesToUpdate, requestContext);

Reply:
thanks, applied